File: FinStack.Domain/Entities/Job.cs
namespace FinStack.Domain.Entities
{
    public class Job
    {
        public int Id { get; set; }
        public Guid Guid { get; set; }
        public string JobType { get; set; }
        public bool Success { get; set; }
        public long ElapsedMs { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime FinishTime { get; set; }
        public string Message { get; set; } = string.Empty;

        public Job()
        {
            Guid = Guid.NewGuid();
        }
    }
}

File: FinStack.Engine/src/models/entities/job.rs
use chrono::{DateTime, Utc};
use sqlx::{Encode, FromRow};
use uuid::Uuid;

#[allow(dead_code)]
#[derive(Debug)]
pub struct Job {
    pub id: i32,
    pub guid: Uuid,
    pub job_code: String,
    pub elapsed: i64,
    pub success: bool,
    pub start_time: DateTime<Utc>,
    pub finish_time: DateTime<Utc>,
    pub message: String,
}

#[derive(Debug, Encode, FromRow)]
pub struct JobDto {
    pub id: i32,
    pub guid: Uuid,
    pub job_code: String,
    pub elapsed: i64,
    pub success: bool,
    pub start_time: DateTime<Utc>,
    pub finish_time: DateTime<Utc>,
    pub message: String,
}
